packages/property-grid/package.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "@bentley/property-grid-react",
+  "version": "1.0.0",
+  "description": "Property Grid React",
+  "license": "MIT",
+  "main": "lib/property-grid-react.js",
+  "typings": "lib/property-grid-react",
+  "scripts": {
+    "copy:assets": "cpx \"./src/**/*.scss\" ./lib && cpx \"./public/**/*\" ./lib/public",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
+    "build": " tsc 1>&2 && npm run copy:assets && npm run pseudolocalize",
+    "clean": "rimraf lib",
+    "lint": "tslint -p . 1>&2",
+    "test": "",
+    "cover": ""
+  },
+  "keywords": [
+    "Bentley",
+    "Property Grid React",
+    "iModel Property Grid",
+    "iModel.js"
+  ],
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/property-grid"
+  },
+  "peerDependencies": {
+    "@bentley/bentleyjs-core": "^2.0.0",
+    "@bentley/context-registry-client": "^2.0.0",
+    "@bentley/frontend-authorization-client": "^2.0.0",
+    "@bentley/geometry-core": "^2.0.0",
+    "@bentley/imodelhub-client": "^2.0.0",
+    "@bentley/imodeljs-common": "^2.0.0",
+    "@bentley/imodeljs-frontend": "^2.0.0",
+    "@bentley/presentation-components": "^2.0.0",
+    "@bentley/imodeljs-i18n": "^2.0.0",
+    "@bentley/imodeljs-markup": "^2.0.0",
+    "@bentley/imodeljs-quantity": "^2.0.0",
+    "@bentley/itwin-client": "^2.0.0",
+    "@bentley/orbitgt-core": "^2.0.0",
+    "@bentley/presentation-common": "^2.0.0",
+    "@bentley/presentation-frontend": "^2.0.0",
+    "@bentley/product-settings-client": "^2.0.0",
+    "@bentley/ui-abstract": "^2.0.0",
+    "@bentley/ui-components": "^2.0.0",
+    "@bentley/ui-core": "^2.0.0",
+    "@bentley/ui-framework": "^2.0.0",
+    "@bentley/ui-ninezone": "^2.0.0",
+    "@bentley/webgl-compatibility": "^2.0.0"
+  },
+  "devDependencies": {
+    "@bentley/bentleyjs-core": "^2.0.0",
```

Comment:
core bentley imjs dev deps shouldnt be using `^` and should be pinned down

Reply:
sorry, just had a discussion with @calebmshafer and jason, and we're going to use `^` for core imjs deps as well now. please ignore my earlier comment, and revert that change